NEWS: Add Adibou2/Sciences testing announcement (#498)
scummvm/scummvm-web: data/en/news/20250624.markdown

---
title: Off to Adibou's Land of Sciences!
author: sdelamarre
date: 1750759200 
---

We are happy to announce that ScummVM now supports _Adibou 2: Nature & Sciences_, an expansion of _Adibou 2_ released in 1998 by Coktel Vision.

> While the original _Adibou 2_ game focused on reading and mathematics, _Nature & Sciences_ explores new topics: history, geography, science & technology, the countryside, and everyday life. Each subject features six playful educational activities. Please note that the 3D car driving minigame, included in the original PC version, is unfortunately not supported in our ScummVM reimplementation, as it was based on a different engine. 

Author: Here I used "PC" to say "Windows", because that non-supported minigame existed only in the Windows version (and I thought "PC" implied "not Mac"). But that piece of information is a detail actually, I'll just remove that part between commas.

> To help us during the testing period, dig out your _Nature & Sciences_ CD, grab a [daily build](https://scummvm.org/downloads/#daily) and read our [testing guidelines](https://wiki.scummvm.org/index.php/Release_Testing#Testing_Guidelines). You will also need an existing installation from the base _Adibou 2: Environment_ CD, and to copy the contents of your _Nature & Sciences_ CD into a freely named subdirectory of the main game directory, as described in [our wiki](https://wiki.scummvm.org/index.php/Adibou_2/). Then, register the expansion in-game via the "Land of Knowledge" screen, and start collecting sweets as you complete Adibou's activities…or encourage a young relative to do so! Hardworking players may even get a diploma upon completing a module and proudly display it in their room!

Reviewer: The last sentence sounds a bit awkward to me. Maybe it can be rephrased?

```suggestion
To help us during the testing period, dig out your _Nature & Sciences_ CD, grab a [daily build](https://scummvm.org/downloads/#daily) and read our [testing guidelines](https://wiki.scummvm.org/index.php/Release_Testing#Testing_Guidelines). You will also need an existing installation from the base _Adibou 2: Environment_ CD, and to copy the contents of your _Nature & Sciences_ CD into a freely named subdirectory of the main game directory, as described in [our wiki](https://wiki.scummvm.org/index.php/Adibou_2/). Then, register the expansion in-game via the "Land of Knowledge" screen, and start collecting sweets as you complete Adibou's activities…or encourage a young relative to do so! Hardworking players may even earn a diploma upon completing a module, which they can proudly display in their room!
```
